Makefile

```diff
@@ -22,14 +22,14 @@ LD_FLAGS = -ldflags " \
 release:                        ## Build pmm-admin release binary.
 	env CGO_ENABLED=0 go build -v $(LD_FLAGS) -o $(PMM_RELEASE_PATH)/pmm-admin
 
-init:                           ## Installs tools to $GOPATH/bin (which is expected to be in $PATH).
-	curl https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(GOPATH)/bin
+init:                           ## Installs development tools
+	go build -modfile=tools/go.mod -o bin/go-consistent github.com/quasilyte/go-consistent
+	go build -modfile=tools/go.mod -o bin/golangci-lint github.com/golangci/golangci-lint/cmd/golangci-lint
+	go build -modfile=tools/go.mod -o bin/reviewdog github.com/reviewdog/reviewdog/cmd/reviewdog
+	go build -modfile=tools/go.mod -o bin/goimports golang.org/x/tools/cmd/goimports
 
-	go install ./vendor/github.com/quasilyte/go-consistent \
-				./vendor/golang.org/x/tools/cmd/goimports
-
-	go test -i ./...
-	go test -race -i ./...
+	go install ./...
+	go test -race ./...
```

Comment:
Oh, I got it. 
```

Reply:
@Dasio ^

Reply:
Sure, go install was needed because of go test was using built binary.
If i remove test from here, it's fine